### ISSUE TITLE
Cannot find formatter warnings

### ISSUE THREAD

**Issue by nullsage** [wix/stylable#848]
There are a lot of cannot find formatter warnings happening when i compile my project.

`format`
`constant`
`env`
`local`

example:
```css
    @font-face {
        font-family: 'Material Icons';
        font-style: normal;
        font-weight: 400;
        src: local('Material Icons'), local('MaterialIcons-Regular'), 
        url(/material-design-icons/iconfont/MaterialIcons-Regular.woff2) format('woff2'), 
        url(/material-design-icons/iconfont/MaterialIcons-Regular.woff) format('woff'), 
        url(/material-design-icons/iconfont/MaterialIcons-Regular.ttf) format('truetype');
}
```

I did not see anything in the docs about formatting these or not using these. Will this be fixed in the near future?

**Comment by tomrav** [comment]
The warning you received refers to the `formatter` feature where you can use custom JS functions in your stylesheets. We're issuing this warning whenever we encounter an unknown function not listed in our CSS native functions list.

I found references for `format`, `env` and `local` and we should add them to said list, so that the warnings will stop.

As for `constant`, it seems like it was renamed to `env` before the spec was finalized? Are you using it differently somehow?

**Comment by nullsage** [comment]
@tomrav Thank you for your quick response. 

We have to support multiple web browsers, of which, IE is one of them. IE currently does not support env() also earlier versions of safari, according to https://developer.mozilla.org/en-US/docs/Web/CSS/env, so the fallback is constant() method as far as i know.

**Comment by tomrav** [comment]
Sounds like a good enough reason for me to add it as well.

**Comment by tomrav** [comment]
Resolved in #858, hope to release it tomorrow - closing the issue.
